ui/package.json

```diff
@@ -106,7 +106,7 @@
     "ember-on-resize-modifier": "^1.0.0",
     "ember-overridable-computed": "^1.0.0",
     "ember-page-title": "^7.0.0",
-    "ember-power-select": "^4.1.7",
+    "ember-power-select": "^7.2.0",
```

Comment:
Note: I could upgrade this to v8+, but that requires adding extra template code per https://ember-power-select.com/docs/migrate-7-0-to-8-0 and I think my time would be better suited moving away from ember-power-select raw usage generally (we use it for our global search, among other things). Helios' dropdown component will be a more than sufficient replacement for it.